src/index.ts

```diff
@@ -264,7 +264,7 @@ export class MessagingExtensionMiddleware implements Middleware {
                         this.processor.onFetchTask) {
                         try {
                             const result = await this.processor.onFetchTask(context, context.activity.value);
-                            const body = result.type === "continue" || result.type === "message" ?
+                            const body = result.type === "message" ?
```

Comment:
Altered this line because it seems `"continue"` is no longer valid for this type:
```
src/index.ts:267:42 - error TS2367: This condition will always return 'false' since the types '"message" | "auth" | "result" | "edit" | "send" | "config" | "botMessagePreview"' and '"continue"' have no overlap.
```

Reply:
Note that `onFetchTask` resolves to either a `MessagingExtensionResult` or a `TaskModuleContinueResponse` so there is still a possibility that `result.type === "continue"` .